### ISSUE TITLE
Crashed the game reviving an AI character

### ISSUE THREAD

**Issue by NilanthAnimosus** [Regalis11/Barotrauma#2006]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
I got bored, shot at an AI with an SMG and revolver seeing how long I could keep them alive full of bullet holes.

First I had healed them when they were dead, then I used the revive command. the error however looks related to the AI's objectives.

**Steps To Reproduce**
Unknown, may be related to  reviving the healed corpse of an AI who had the operate reactor objective before death, doubtful this reproduces however.

**Version**
0.9.6.0

**Additional information**

Error stack:

```
Exception: Object reference not set to an instance of an object.
Target site: Barotrauma.AIObjective CreateObjective(Barotrauma.Order, System.String, Barotrauma.Character, Single)
Stack trace: 
   at Barotrauma.AIObjectiveManager.CreateObjective(Order order, String option, Character orderGiver, Single priorityModifier)
   at Barotrauma.AIObjectiveManager.SetOrder(Order order, String option, Character orderGiver)
   at Barotrauma.HumanAIController.SetOrder(Order order, String option, Character orderGiver, Boolean speak)
   at Barotrauma.Character.SetOrder(Order order, String orderOption, Character orderGiver, Boolean speak)
   at Barotrauma.CrewManager.AddCharacter(Character character)
   at Barotrauma.CrewManager.ReviveCharacter(Character revivedCharacter)
   at Barotrauma.Character.Revive()
   at Barotrauma.DebugConsole.ExecuteCommand(String command)
   at Barotrauma.DebugConsole.Update(Single deltaTime)
   at Barotrauma.GameMain.Update(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.Tick()
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop()
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)
   at Barotrauma.Program.Main(String[] args)
```

Full crashlog: 
[crashreport.log](https://github.com/Regalis11/Barotrauma/files/4080109/crashreport.log)

**Comment by itchyOwl** [comment]
Couldn't repro. Were you perhaps in the freecam mode? There was a bug that caused a crash due to a null reference exception. It was fixed in Regalis11/Barotrauma-development@f8cc2ae and the fix is included in the current unstable build.

**Comment by NilanthAnimosus** [comment]
the full crashreport was posted in the event I did not remember and honestly - I did not but can picture what I did in my head. I think I got bored and just started killing an AI over and over checking gun damage and such. Using fix items to repeatedly repair an SMG magazine, spawning morphine/saline/plastiseal and such and repeatedly killing and reviving a player via CPR and items.

Given I was using fixitems in the console and cannot see an order of freecam, I was most definately controlling a character at the time with medicals and a gun on hand.

seems while likely stacking gunshot wounds I had killed the AI and went to heal then revive them, which then the game crashed.

The last two commands do indicate me healing first though:

Last debug messages:
[18/01/2020 07:21:35] revive steve vazquez
[18/01/2020 07:21:28] heal steve vazquez
